src/main/java/org/example/Main.java

```diff
@@ -28,6 +28,16 @@ public static void main(String[] args) {
             res=num1*num2;
             System.out.println("Результат: "+res);
             break;
+        case "^":
+        System.out.println("Якого числа \"1\" чи \"2\"?: ");
+        int a=scanner.nextInt();
+        if (a==1)
```

Comment:
make separete function for input values step by step, firs i put value1, than valu2